src/transfers-history/services/SpokePoolEventsQueryService.ts

```diff
@@ -23,8 +23,8 @@ export class SpokePoolEventsQueryService {
     let speedUpDepositEvents: RequestedSpeedUpDepositEvent[] = [];
     let blockTimestampMap: { [blockNumber: number]: number } = {};
 
-    if (this.latestBlockNumber) {
-      from = this.latestBlockNumber + 1;
+    if (this.latestBlockSearched) {
```

Comment:
Do we want to check if this is undefined, or is it okay to jump to the else statement if `latestBlockSearched === 0`?

My assumption is that we consider latestBlockSearched=0 an erroneous state, but it's worth double checking